locales/templates/app.pot

```diff
@@ -301,6 +301,26 @@ msgctxt "content_description_erase"
 msgid "Erase browsing history"
 msgstr ""
 
+#. Content description (not visible, for screen readers etc.): A floating
+#. action button that       is shown on top of the web content. Clicking the
+#. button shows (expands) the menu for switching       tabs, adding a new tab
+#. or erasing the browsing history.
+msgctxt "content_description_fab_expand"
+msgid "Expand"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): "Plus" button
+#. for adding a new     temporary tab.
+msgctxt "content_description_open_tab"
+msgid "Open tab"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): "X" button for
+#. closing the      screen to enter a URL for a new tab to be opened.
+msgctxt "content_description_close_tab"
+msgid "Close tab"
```

Comment:
Same here, better comment (I honestly don't understand the current one at all…).

> Description of the "X" button used to close a new temporary tab ("close" is a verb), not displayed and read only by screenreaders.